test/input/yaml/CustomInput_loop_da.yaml

```diff
@@ -0,0 +1,155 @@
+---
+#initial input parameters
+log_parameters:
+    verbose: true  # verbose output (leave blank for quiet output.)
+    showtiming: true  # set the showtiming (omit flag for no timing information.)
+    debuglevel: 1  # set the debuglevel for additional console output.
+network_topology_parameters:
+    supernetwork_parameters:
+        title_string: "Florence_FullRes with Nudging in Forecast Mode"
+        geo_file_path: "/glade/scratch/jamesmcc/tmp_florence_run_nudging/forecasts/cast_2018091800/NWM/DOMAIN/Route_Link.nc"
```

Comment:
We can't have system file paths -- everything has to reference the repository, even if it is a symlinked folder that is broken on another system.